frontend/src/components/DataTable.vue

```diff
@@ -287,27 +259,18 @@ export default defineComponent({
         return true
       })
 
-      if (sort.header) {
+      const sortHeader = sort.header as TableHeader
+      if (sortHeader) {
+        console.time(`${config.settings.translationPrefix} SORT`)
         rows.sort((a : TableRow, b : TableRow) => {
-          let valueA  = a[sort.header.key]
-          let valueB  = b[sort.header.key]
-          const order = sort.order ? -1 : 1
+          const valueA = sortHeader.value(a, sortHeader)
```

Comment:
there could be a confusion between the `value` of a ref and the value function of the header. Maybe rename to `valueFn`

Reply:
So, this is also the function the user will override to provide a custom value  accessor. 

Same as :
- search (TBD)
- format
- compare
- value

The user can either for all of theses provide a string, or a function.

It might make it more clear for the Datatable internals, but less clear for a devloppers nwho use the datatable

Reply:
Ive renamed the only other reference to `value` 
```patch
-      const value = header.value(row, header)
-      return header.format(value, row)
+      const fieldValue = header.value(row, header)
+      return header.format(fieldValue, row)
```

Reply:
:+1: